Source/Particles/Collision/PairWiseCoulombCollision.cpp

```diff
@@ -51,7 +51,7 @@ PairWiseCoulombCollision::doCollisions (amrex::Real cur_time, MultiParticleConta
     for (int lev = 0; lev <= species1.finestLevel(); ++lev){
 
         // Loop over all grids/tiles at this level
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
         info.SetDynamic(true);
```

Comment:
This looks odd  here, I think we set this generally now as an input option.

Reply:
Apparently `info.SetDynamic(true);` is used only in two other places in the whole Warpx source code. I agree with you that it should not be set like this...

Reply:
more updates below